client/src/features/dataConnectorsV2/components/DataConnectorView.tsx

```diff
@@ -218,73 +196,24 @@ function DataConnectorViewConfiguration({
   dataConnector,
 }: Pick<DataConnectorViewProps, "dataConnector">) {
   const storageDefinition = dataConnector.storage;
-  const credentialFieldDefinitions = useMemo(
-    () =>
-      getCredentialFieldDefinitions({
-        storage: storageDefinition,
-        sensitive_fields: storageDefinition.sensitive_fields,
-      }),
-    [storageDefinition]
-  );
-  const requiredCredentials = useMemo(
-    () =>
-      credentialFieldDefinitions?.filter((field) => field.requiredCredential),
-    [credentialFieldDefinitions]
-  );
-  const nonRequiredCredentialConfigurationKeys = Object.keys(
-    storageDefinition.configuration
-  ).filter((k) => !requiredCredentials?.some((f) => f.name === k));
 
   return (
     <section
-      className={cx("border-top", "pt-3")}
+      className={cx("border-top", "border-dark", "pt-3")}
       data-cy="data-connector-configuration-section"
     >
       <div>
-        <h4>Configuration</h4>
+        <h4 className="mb-4">
+          <Gear className={cx("bi", "me-1")} /> Configuration
```

Comment:
This removes the extra space when copy-pasting:

```suggestion
          <Gear className={cx("bi", "me-1")} />
          Configuration
```